inst/as_top_level.R

```diff
@@ -0,0 +1,10 @@
+function(shiny_module) {
```

Comment:
I would suggest adding a comment that explains, why this file is kept in `inst`.